src/features/analytics/analytics-utils.ts

```
@@ -0,0 +1,552 @@
+    case 'last30Days':
+      return 'letzte 30 Tage';
+    case 'last90Days':
+      return 'letzte 90 Tage';
+    case 'thisYear':
+      return 'dieses Jahr';
+    case 'custom':
+      if (!timeframe.from || !timeframe.to) {
+        return 'benutzerdefiniert';
+      }
+
+      const fromDate = parseISO(timeframe.from);
```

Biome (2.4.11)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)